src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -201,6 +206,11 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
                 sorting = ms5.sorting_scheme3(recording=recording_cached, sorting_parameters=scheme3_sorting_parameters)
             else:
                 raise ValueError(f"Invalid scheme: {scheme} given. scheme must be one of '1', '2' or '3'")
+        except Exception as e:
+            print(f"An error occurred: {e}")
+
+        if tmpdir and os.path.isdir(tmpdir.name):
+            shutil.rmtree(tmpdir.name, ignore_errors=True)
```

Comment:
For this I mean because the tmp file can actually be pretty hidden in a file tree and could fill up the user's computer. So if we fail to clean we need to tell the user where to go to clean.